app/models/ext_management_system.rb

```diff
@@ -66,6 +66,7 @@ def self.provider_create_params
            :inverse_of => :ext_management_system
   has_many :miq_templates,     :foreign_key => :ems_id, :inverse_of => :ext_management_system
   has_many :vms,               :foreign_key => :ems_id, :inverse_of => :ext_management_system
+  has_many :orchestration_stacks, :foreign_key => :ems_id, :dependent => :nullify, :inverse_of => :ext_management_system
```

Comment:
@bzwei Can you verify whether these should be `:dependent => :nullify` or something else?

Reply:
adding `nullify` seems reasonable.

Reply:
The mixin declares it as `dependent => :destroy`

Reply:
Any reason this relation is declared in a mixin rather than in the base class?